hack/update-openapi-spec.sh

```diff
@@ -47,20 +47,20 @@ trap cleanup EXIT SIGINT
 
 kube::golang::setup_env
 
-TMP_DIR=$(mktemp -d /tmp/update-openapi-spec.XXXX)
+TMP_DIR=${TMP_DIR:-$(kube::realpath "$(mktemp -d -t "$(basename "$0").XXXXXX")")}
```

Comment:
If you want to scor some more cleanup points, all of these should be quoted:

```
ETCD_HOST="${ETCD_HOST:-127.0.0.1}"
...
API_LOGFILE="${API_LOGFILE:-${TMP_DIR}/openapi-api-server.log}"
```

etc :)